webpack/manifest.ts

```diff
@@ -139,6 +139,12 @@ const buildManifestCommons = (vendor: string): Omit<Manifest.WebExtensionManifes
         js: ['scripts/contentScript.js'],
         run_at: 'document_start',
         all_frames: true
+      },
+      {
+        matches: ['https://etherscan.io/*', 'https://bscscan.com/*', 'https://polygonscan.com/*'],
```

Comment:
Not to forget: we must ensure that this permissions change won't automatically disable extensions for our users.